consumer/src/main/kotlin/streams/kafka/KafkaEventSink.kt

```diff
@@ -148,7 +147,7 @@ class KafkaEventSink(private val config: Config,
                             }
                             queryExecution.writeForTopic(topic, data)
                         }
-                        TimeUnit.SECONDS.toMillis(1)
+                        streamsConfig.pollInterval
```

Comment:
Below, you set a new default poll interval of 100ms, where it looks like the old one was 1ms.  Are you sure this is a better new default?  (I'm not sure myself, I just mean this introduces potential latency, but also avoids "busy waiting"). 

Wondering if you saw in the flamegraph that too much busy waiting was happening here and that's why this change, but I'm sensitive to changes in the default, because they can affect other workflows.  If you've got a specific reason then it's ok

Reply:
No the 100ms is a random number for the tests, I defined the new one a 0 that should work better as default.